openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -347,6 +347,13 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
                               ssl_context_cleanup,
                               apr_pool_cleanup_null);
 
+
+#ifndef OPENSSL_NO_TLS1_3
+    // We specify this as we currently not handle these extra tickets send after the handshake / shutdown
+    // in netty. We may want to remove this call once we do.
+    SSL_CTX_set_num_tickets(ctx, 0);
```

Comment:
I suspect this is not needed anymore after my recent changes. I will check this later today as the wifi on the plane is not really great for downloading openssl 1.1.1 ;)